src/prebid.js

```diff
@@ -387,29 +399,55 @@ pbjs.removeAdUnit = function (adUnitCode) {
   }
 };
 
+pbjs.clearAuction = function() {
+  auctionRunning = false;
+  utils.logMessage('Prebid auction cleared');
+};
+
 /**
  *
  * @param bidsBackHandler
  * @param timeout
+ * @param adUnits
+ * @param adUnitCodes
  */
-pbjs.requestBids = function ({ bidsBackHandler, timeout }) {
+pbjs.requestBids = function ({ bidsBackHandler, timeout, adUnits, adUnitCodes }) {
+  if (auctionRunning) {
+    utils.logError('Prebid Error: `pbjs.requestBids` was called while a previous auction was' +
+      ' still running. Resubmit this request.');
+    return;
+  } else {
+    auctionRunning = true;
+    pbjs._bidsRequested = [];
+    pbjs._bidsReceived = [];
+    resetPresetTargeting();
```

Comment:
@protonate 
I think it makes sense to move this into the `setTargetingForGPTAsync` function, because this is GPT specific code. Thoughts?